init/retina/main_linux.go

```diff
@@ -21,23 +22,25 @@ var (
 func main() {
 	// Initialize logger
 	opts := log.GetDefaultLogOpts()
-
+	zl, err := log.SetupZapLogger(opts)
+	if err != nil {
+		panic(err)
+	}
+	l := zl.Named("init-retina").With(zap.String("version", version))
+	// Load configuration
+	cfg, err := config.GetConfig("/retina/config/config.yaml")
```

Comment:
i don't think we should be hardcoding config file path

Reply:
I use the `configPath := flag.String("config", "/default/path/to/config.yaml", "path to the config file")` to replace the hardcoding. Do you think it ok?

Reply:
yes, that is fine, but instead of `"/default/path/to/config.yaml"` you can use `/retina/config/config.yaml` since it is the default directory for config by default

Reply:
Thank you for your help. I wasn't sure about the file path, so I changed it as you suggested.